x-pack/platform/plugins/shared/ml/public/application/jobs/jobs_list/jobs.tsx

```diff
@@ -43,12 +44,19 @@ export const JobsPage: FC<JobsPageProps> = ({ isMlEnabledInSpace, lastRefresh })
     getDefaultAnomalyDetectionJobsListState()
   );
   const {
-    services: { docLinks },
+    services: { docLinks, spaces },
   } = useMlKibana();
   const { euiTheme } = useEuiTheme();
 
   const { showNodeInfo } = useEnabledFeatures();
   const helpLink = docLinks.links.ml.anomalyDetection;
+
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const ContextWrapper = useCallback(
```

Comment:
We have a mixture of different ways of creating a space management context wrapper in the our code now. some doing it this way, some using `useSpacesContextWrapper`.
We could create a single wrapping component which uses `useSpacesContextWrapper` and wraps it around the children passed in and use that everywhere.
something like

```ts
export const SpacesContextWrapper: FC<PropsWithChildren<unknown>> = ({ children }) => {
  const ContextWrapper = useSpacesContextWrapper();

  return <ContextWrapper>{children}</ContextWrapper>;
};
```

Reply:
Updated here [`65b457b` (#206696)](https://github.com/elastic/kibana/pull/206696/commits/65b457bf12ce7c01e88befe195dc55aa86ad6cd2)

Reply:
I'm seeing these errors in the console when loading the jobs or models lists pages.

![image](https://github.com/user-attachments/assets/f3482497-4917-4799-959a-a39fa81360b7)